test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -860,10 +860,11 @@ public class ModelBindingTest
             // Assert
             var json = JsonConvert.DeserializeObject<Dictionary<string, string>>(response);
             Assert.Equal(3, json.Count);
+
             // Mono issue - https://github.com/aspnet/External/issues/19
-            Assert.Equal(PlatformNormalizer.NormalizeContent("The Field1 field is required."), json["Field1"]);
```

Comment:
Suggest adding a comment explaining why the prefix is used in the errors e.g. `// Prefix exists in value providers but requested values do not.`